scripts/buildPackage.js

```diff
@@ -157,5 +158,7 @@ const createESMRawBuild = async () => {
   await buildProd(rawConfigChunks);
 };
 
-createESMRawBuild();
-createESMBrowserBuild();
+(async () => {
+  await createESMRawBuild();
+  await createESMBrowserBuild();
+})();
```

Comment:
btw node.js should wait for this before script exists, but not sure if guaranteed. Should we make this into an `mjs` script and use top-level await?

Reply:
actually it should not be really needed looking at the change, just added it defensively as the build was failing from time to time; will revert and we'll see what happens